### ISSUE TITLE
Hello, I have read your SOT code carefully, there is a question to ask you, how to add loss function to make the target domain samples to achieve better classification effect? Is there a code or related papers recommended to me

### ISSUE THREAD

**Issue by maoshiyun** [jindongwang/transferlearning#368]
**Before you open an issue**
If it's code running error, maybe you want to check the python or pytorch version before submitting an issue.

**Describe the bug**

**To Reproduce**
Where is this bug happen?

**Screenshots**

**Comment by jindongwang** [comment]
@lw0517

**Comment by jindongwang** [comment]
@maoshiyun This is not an issue. Designing better algorithms to improve accuracy should be more research-oriented and this codebase will not cover. I will close this.
